digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/repository/SyncLogQueryBuilder.java

```diff
@@ -1,7 +1,9 @@
 package org.digit.health.sync.repository;
 
-import org.digit.health.sync.web.models.request.SyncLogSearchDto;
+import org.digit.health.sync.web.models.dao.SyncLogData;
 
 public interface SyncLogQueryBuilder {
-    String getSQlBasedOn(SyncLogSearchDto syncLogSearchDto);
+    String getSQlBasedOn(SyncLogData syncLogSearchDto);
```

Comment:
variable name should also be syncLogData

Reply:
done